Owncloud iOs Client/Network/Uploads/ManageUploadRequest.m

```diff
@@ -491,7 +488,7 @@ - (void)finishOverwriteProcess{
     if (!IS_IPHONE){
         [app.detailViewController presentWhiteView];
         //Launch a notification for update the previewed file
-        [[NSNotificationCenter defaultCenter] postNotificationName:fileDeleteInAOverwriteProcess object:self.currentUpload.destinyFolder];
+        [[NSNotificationCenter defaultCenter] postNotificationName:FileDeleteInAOverwriteProcess object:self.currentUpload.destinyFolder];
```

Comment:
Probably you should remove this observers: FileDeleteInAOverwriteProcess, UploadOverwriteFileNotification, PreviewFileNotificationUpdated